### DISCUSSION TITLE
Is it possible to invert or remap midi values reiceved by midi2lr ?

### DISCUSSION THREAD

**Discussion by iprion** [rsjaffe/MIDI2LR#733] [Q&A]
For example when I turn a knob clockwise controlling "Hue on global color grading", Hue is increasing but in Lightroom UI Hue cursor is turning counter clockwise.

**Comment by paul-cine** [comment]
Right click on a mapping and reverse the numbers 127 - 0

**Reply by RBCAMERA** [reply]
I tried this but it does not work Midi2LR defaults back..

**Reply by paul-cine** [reply]
yeah you're right. i am pretty sure this worked in the past.  Afraid the only way is either changing the behaviour on the controller itself if it can be programmed, otherwise you need to use a app like Bome convert the midi message from controller to Bome, then from Bome to Midi2LR.
